### ISSUE TITLE
Cydia always crashes on ipad 5th

### ISSUE THREAD

**Issue by kimsh22** [Sticktron/g0blin#266]
Sydia did not work on rc1. So I jailbreak again with rc2, but Sydia crashes again on Ipad5th.

**Comment by andrewathalye** [comment]
iPad 5 as in iPad Air ? Try rebooting and using the reinstall cydia option.

**Comment by kimsh22** [comment]
I mean ipad 2017. I tried all methods like resinstall option, using meridian to remove and reinstall the Cydia. I think g0blin has bug that cydia crashes on ipad 2017 ios 10.3.1. I have seen Cydia works with 10.3.3 but have not seen one that worked on 10.3.1.

**Comment by andrewathalye** [comment]
What is an iPad 2017? That's not an official product name ...

Le 19 févr. 2018 à 07:32, kimsh22 <notifications@github.com<mailto:notifications@github.com>> a écrit :


I mean ipad 2017. I tried all methods like resinstall option, using meridian to remove and reinstall the Cydia. I think g0blin has bug that cydia crashes on ipad 2017 ios 10.3.1. I have seen Cydia works with 10.3.3 but have not seen one that worked on 10.3.1.

**Comment by Sticktron** [comment]
iPad 5G == 2017 iPad

**Comment by andrewathalye** [comment]
Ah. Is that even supported?

**Comment by Sticktron** [comment]
Yes it is, although it's possible the offsets could be wrong for a particular version of iOS.

**Comment by andrewathalye** [comment]
I guess...or maybe there isn't enough ram or flash memory left?

**Comment by fabio-magalhaes** [comment]
Hi, yes, my iPad is 5G and it´s in 10.3.2, but sometimes a need try a lot times.
